### ISSUE TITLE
Surface generation errors

### ISSUE THREAD

**Issue by vistaero** [aeroson/UnityProceduralPlanets#1]
I think these images are self-explanatory:

![bug1](https://user-images.githubusercontent.com/4535360/35976756-64ed8fb4-0ce1-11e8-845e-4e59a4fc6772.PNG)
![bug2](https://user-images.githubusercontent.com/4535360/35976758-650ffc66-0ce1-11e8-81e3-675a48d7830a.PNG)
![bug3](https://user-images.githubusercontent.com/4535360/35976759-653301b6-0ce1-11e8-9350-69f6fa347809.PNG)

**Comment by JakubNei** [comment]
Did you increase the planet radius ? If it's over 10k it will break.
Also I tried to improve it a little: 68f356db2f785c20bbed3d3c98271fdce79fcd9f

**Comment by vistaero** [comment]
Yes, I wanted the radius of the Earth. Could this issue be avoided if we give it a premade height map of the Earth? http://www.celestiamotherlode.net/catalog/earthbumpspec.php here there are 8K and 32K versions. I already tried but I can't see any continents even commenting out GeneratePlanetData()

Thanks for your work, I hope to use this to make someday the funniest solar system simulator :D

**Comment by JakubNei** [comment]
Premade earth height map won't help, since the issue is probably in the float calculations of height map & normal map of chunks, such as dividing 10k by 0.0001.
I need to figure out a different, better way to calculate those.
